cs3api4lab/auth/reva_token.py

```diff
@@ -0,0 +1,18 @@
+from tornado import web
+
+from cs3api4lab.auth.authenticator import Authenticator
+
+
+class RevaTokenAuth(Authenticator):
```

Comment:
I don't see a reason for having this atm.

Reply:
The WOPI has two types of auth with user/pass and token (UserID in method). Now I can't say if token auth will be using, but another project has this functionality. Example: https://github.com/cs3org/wopiserver/blob/0b6f9198e5d3629e3d5903b305ad9b85d51e628b/src/cs3iface.py#L40

Reply:
I don't understand. The link you sent if for testing and is username/password. But still, for now I don't see a need to have reva tokens because we're not being called by reva.

Reply:
I removed Riva token auth method.